mycore.org/content/de/documentation/search/search_solr_use.html

```diff
@@ -1,835 +1,1040 @@
+<p>
+    Zunächst muss die Solr-Server-URL der MyCoRe-Anwendung über ein Property bekannt
+    gemacht werden (siehe <a href="{{< ref gs_properties >}}">MyCoRe-Konfigurationsverzeichnis</a>).
+    Weiterhin können für die einzelnen <em>CoreIds</em> der <em>CoreName</em> und falls abweichend auch die URLs
+    definiert werden.
+    Die <em>CoreId</em> <code>main</code> ist dabei immer erforderlich. Weitere <em>CoreIds</em> wie
+    <code>classification</code> sind optional.
+
+    Es ist auch möglich, den Core auf einem anderen Solr Server zu erstellen. In diesem Fall
+    muss zusätzlich das Property <code>MCR.Solr.Core.{core_id}.ServerURL</code>
+    gesetzt werden. Somit können Solr-Cores auch auf verschiedenen physischen Hosts liegen.
+</p>
+
+{{< highlight shell "linenos=table">}}
+MCR.Solr.ServerURL=http://localhost:8983/
+MCR.Solr.Core.main.Name=my_core_name
+MCR.Solr.Core.main.ServerURL=%MCR.Solr.ServerURL%
+...
+{{< /highlight >}}
+
+<p>
+    Alternativ können die <em>CoreIds</em> über die MyCoRe-CLI registriert werden. Diese Einstelllungen sind jedoch
+    nicht persistent.
+</p>
+
+<p>
+    Bis MyCoRe 2023.06 LTS:
+</p>
+{{< highlight shell "linenos=table">}}
+    register solr core with name {core_name} as core main
+    register solr core with name my_core_name as core main
+{{< /highlight >}}
+
+
+<p>
+    Ab MyCoRe 2024.06 LTS:
+</p>
+{{< highlight shell "linenos=table">}}
+    register solr core with name {0} as core {1}
+    add type {0} to solr core {1}
+
+    register solr core with name my_core_name as core main
+    add type main to solr core main
+{{< /highlight >}}
+
+
+<h3>Reguläre Solr-Installation</h3>
+
+<h4>Configuration Sets laden</h4>
+<p>
+    Die Basis-Konfiguration eines Solr-Cores erfolgt über <strong>Configuration Sets</strong>.
+    Die Konfigurationen sind je nach MyCoRe-Version unterschiedlich. Ab 2024.06 LTS werden die Konfigurationen direkt
+    im Modul <code>mycore-solr</code> bereitgestellt. Für ältere Versionen müssen die Konfigurationen aus einen anderen
+    Repository geladen werden.
+<p>
+<h5>bis MyCoRe 2023.06 LTS</h5>
+<p>
+    Bei GitHub werden verschiedene
+    <a href="https://github.com/search?q=topic%3Asolr-core+org%3AMyCoRe-Org">MyCoRe-Config-Sets</a>
+    zur Verfügung gestellt. Für den Solr-Core vom Typ <code>main</code> wird das
+    <a href="https://github.com/MyCoRe-Org/mycore_solr_configset_main">mycore_solr_configset_main</a>
+    benötigt.
+</p>
+<h5>ab MyCoRe 2024.06 LTS</h5>
+<p>
+    Die <a href="https://github.com/MyCoRe-Org/mycore/tree/main/mycore-solr/src/main/resources/configset">MyCoRe-Config-Sets</a>
+    sind direkt im Modul <code>mycore-solr</code> enthalten.
+</p>
+<p>
+    Um das Set in den Solr-Server zu integrieren, können folgende Kommandos ausgeführt werden:
+</p>
+
+{{< highlight shell "linenos=table">}}
+cd .../solr/server/solr/configsets
+git clone https://github.com/MyCoRe-Org/mycore_solr_configset_main.git mycore_main
+{{< /highlight >}}
+
```

Comment:
Das gilt auch nur bis 2023.06 LTS, danach müsste man die Configsets aus `mycore-solr/src/main/resources/configset` in den SOLR-Ordner kopieren, oder?